stress_test.py

```diff
@@ -267,15 +278,19 @@ def main():
     if args.heavy:
         print('Setting the heavy configuration...')
         args.large_file_size_gb = 20
-        args.bundle_upload_count = 2000
-        args.create_worksheet_count = 2000
-        args.parallel_runs_count = 1000
-        args.large_docker_runs_count = 100
+        args.bundle_upload_count = 1000
+        args.create_worksheet_count = 1000
+        args.parallel_runs_count = 500
+        args.large_docker_runs_count = 2000
```

Comment:
Changed from 100 to 2000 because previously we used to cycle through the list of 10 Docker example images 100 times, but now instead we randomly pick an image for 2000 runs.

Reply:
@teetone What's the point of randomly picking a Docker image? It makes the test non-deterministic, which we might want to avoid. (if one of the docker images suddenly stops working / breaks, we would rather that that causes a deterministic failure of our tests than a non-deterministic failure)

Reply:
@epicfaace In the previous version of the test, It will just cycle through the list of large Docker images. `docker_image_manager.py` uses an LRU cache to store the Docker images. With an LRU cache, it will simply kick out the images earlier in the list, dealing with the same sequence of image sizes over and over again. Now if we start picking images at random, the LRU cache will have to deal with varying sizes of Docker images coming in, which better mimics real-world behavior.

To your second point, we select an image from a list of 20 images uniformly at random 2000 times. Each image in the list should be used multiple times. If a specific image is problematic, it will be caught by this stress test and fail. Also, our main concern and the point of the stress test is to see if our system can create a bunch of bundles with large Docker images and not if a certain image causes failures or not.